appholder/src/main/java/com/android/identity/wallet/composables/DocumentValues.kt

```diff
@@ -14,7 +14,7 @@ import com.android.identity.wallet.theme.YellowGradient
 fun keystoreNameFor(implementation: SecureAreaImplementationState): Int {
     return when (implementation) {
         is SecureAreaImplementationState.Android -> R.string.keystore_android
-        is SecureAreaImplementationState.BouncyCastle -> R.string.keystore_bouncy_castle
+        is SecureAreaImplementationState.Software -> R.string.keystore_software
     }
```

Comment:
I don't see how this is going to work, I think `SecureAreaImplementationState` will need to carry the name of the Secure Area...